docs/tutorials/light-up.md

```diff
@@ -0,0 +1,224 @@
+Select the path to the camera you want to use.
+Often on Linux systems, this is `video0` or `video1`.
+On Mac, this is often a long string of letters and numbers.
+
+Click **Save Config** in the bottom left corner of the screen.
+
+Navigate to the **CONTROL** tab where you can see your camera working.
+
+{{< gif webm_src="../img/light-up/control-panel.webm" mp4_src="../img/light-up/control-panel.mp4" alt= "Opening the camera panel on the control tab, toggling the video feed on, and watching as a person wearing headphones waves at the camera." >}}
+
+## Set up the Kasa Smart Plug
+
+Plug your smart plug into any power outlet and turn it on by pressing the white button on the smart plug.
+To connect the plug to your wifi, download the Kasa Smart app from the [App Store](https://apps.apple.com/us/app/kasa-smart/id1034035493) or [Google Play](https://play.google.com/store/apps/details?id=com.tplink.kasa_android&gl=US) to your mobile phone.
+When you first open the app, you will be prompted to create an account.
+As you do this, you will receive an email with subject line "TP-Link ID: Activation Required" to complete your account registration.
+
+Next, follow the steps in Kasa's [How to set up my TP-Link Smart Plug Switch via Kasa guide](https://www.tp-link.com/us/support/faq/946/) to add your device and connect it to your wifi.
+Once it is connected, you will no longer need to use the mobile app.
+
+Next, open a terminal on your computer and run the following command to download the [smart plug Python API](https://github.com/python-kasa/python-kasa):
+
+```bash
+pip3 install python-kasa
+```
+
+Next, run the following command to return information about your smart device:
+
+```bash
+kasa discover
+```
+
+You should see this command output something like this:
+
+![Terminal output with information about the smart plug including the host, device state (on), timestamp, hardware and software versions, MAC address, location (latitude and longitute), whether the LED is currently on, and the timestamp of when it last turned on. There is also a list of modules (schedule, usage, antitheft, time and cloud).](../img/light-up/kasa-discover-output.png)
+
+Copy the host address (for example, 10.1.11.221).
+You will need to include it in your Python code in a later step.
+
+## Write Python code to control your object detection robot
+
+Navigate to the **CODE SAMPLE** tab on the Viam app.
+Make sure **Python** is selected in the **Language** selector.
+
+Click the **COPY CODE** button.
+Paste the code into a new Python file on your computer.
+This code connects your script to your robot.
+
+Now you are ready to write the code to turn your smart plug on and off based on camera readings.
+You will use a [TFLite](https://www.tensorflow.org/lite) model to detect specific objects, and a corresponding text that holds class labels for your TFLite model.
+For more information on the Vision Service, check out our [documentation](/services/vision/).

```

Comment:
I think move this last line to later when you've actually mentioned the vision service. This intro could generally be better